packages/client/tests/functional/chunking-query/tests.ts

```diff
@@ -0,0 +1,265 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient, Tag } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite(
+  ({ provider, providerFlavor }) => {
+    function generatedIds(n: number) {
+      // ["1","2",...,"n"]
+      const ids = Array.from({ length: n }, (_, i) => i + 1)
+      return ids
+    }
+
+    // Note: we're excluding `sqlite` from the success cases until `createMany` is added
+    // to the SQLite driver. See https://github.com/prisma/prisma/issues/10710.
+    // We're also excluding `js_neon` the `createMany` would take ages otherwise (while `js_pg` is much).
```

Comment:
Why does it takes ages with Neon?
What is the `(while `js_pg` is much)` supposed to mean? Don't understand.

Reply:
Where are we actually exclusing `js_neon`? Just outdated comment?

Reply:
The comment for `js_neon` no longer applies. I had added the comment in a previous iteration, where my environment loaded Node.js rather than Node.js 20. This resulted in WebSockets silently failing to connect to the database, resulting in a stalled CI (until the 180s time limit of our test suite is reached).
I'll remove the comment, thanks for spotting it out.

Note: I've double-checked that the CI for Driver Adapters uses Node.js 20 only.